### ISSUE TITLE
[Feature Request] Add "PrinterPaperNames" property in getPrinters and getDefaultPrinter methods

### ISSUE THREAD

**Issue by smallsoft-gustavo** [artiebits/pdf-to-printer#446]
**Is your feature request related to a problem? Please describe.**
I'm building a layout builder for printers and I'd like to list the printer available papers to user select.

**Describe the solution you'd like**
I'd like to both methods getPrinters and getDefaultPrinter return PrinterPaperNames property.

**Describe alternatives you've considered**
Option 1: List of properties in method argument, for example: I want to see printer paper names and check the printer availability. something like ['PrinterPaperNames', 'Availability'] and so on.

Option 2: Just pass the property hardcoded "PrinterPaperNames" to Win32_Printer command or maybe

Resolution: 
https://github.com/artiebits/pdf-to-printer/blob/0ecaa3f05b01877e8d88502fc965b28cd8eee95d/src/get-default-printer/get-default-printer.ts#L14-L17

Will be something like:

```
// getDefaultPrinter()
const { stdout } = await execFileAsync("Powershell.exe", [
    "-Command",
    "Get-CimInstance Win32_Printer -Property DeviceID,Name,PrinterPaperNames -Filter Default=true",
  ]);

// and to getPrinters()
const { stdout } = await execFileAsync("Powershell.exe", [
  "-Command",
  "Get-CimInstance Win32_Printer -Property DeviceID,Name,PrinterPaperNames",
]);
```

and of course type the Printer interface

https://github.com/artiebits/pdf-to-printer/blob/0ecaa3f05b01877e8d88502fc965b28cd8eee95d/src/get-default-printer/get-default-printer.ts#L5-L8

```
export interface Printer { 
   deviceId: string; 
   name: string;
   printerPaperNames: string[]
 } 
```

**Additional context**
Will be nice if the request were applied to both 'getPrinters' and getDefaultPrinter' methods.

I can contribute to a PR if needed.

> Sorry for my english :/

**Comment by artiebits** [comment]
hi @smallsoft-gustavo. great idea, please feel free to contribute to this library, it would be appreciated!

**Comment by smallsoft-gustavo** [comment]
I just opened a PR https://github.com/artiebits/pdf-to-printer/pull/447

**Comment by artiebits** [comment]
I am closing the issue because is has been resolved with #447
